search/templates/search/home.html

```diff
@@ -1,11 +1,13 @@
 {% extends "index.html" %}
 {% load static %}
+{% load cache %}
 
 {% block header_tags %}
 <link rel="stylesheet" href="{% static 'css/swimming-fish.css' %}">
 {% endblock %}
 
 {% block content %}
+{% cache 3600 home_page_intro user.is_authenticated %}
```

Comment:
@autn this will cache the content for all authenticated users to be the same. Is that what we want. I'm pretty sure that on JSOS I was careful that the things that were cached like this were OK to be cached the same for all anonymous and logged-in users.

Does that make sense? I.e., on JSOS, it's caching whether the button says 'login' which is the same for all anonymous users, or 'logout' which is the same for all logged-in users. It doesn't cache anything that is unique to each logged-in user. If it did that it would leak user information across users.

Reply:
@YPCrumble Yep, it's exactly. So what is the thing we should change? This block does not include user information currently. Or do you want to prevent the risks in the future?